crates/fluvio/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "fluvio"
-version = "0.14.0"
+version = "0.14.2"
```

Comment:
No need to bump since `0.14.0` is not published yet.

Reply:
fixed